scratch-vm/src/extensions/noise/index.js

```
@@ -0,0 +1,200 @@
+
+(async () => {
+    const module = await Scratch.external.importModule(
+        "https://cdn.jsdelivr.net/npm/fastnoise-lite@1.1.1/FastNoiseLite.min.js",
+    );
+
+    const FastNoiseLite = module.default;
+
+    const noise = new FastNoiseLite();
+
+    const NOISE_TYPES = Object.values(FastNoiseLite.NoiseType);
+    const FRACTAL_TYPES = Object.values(FastNoiseLite.FractalType);
```

ESLint

[error] 'FRACTAL_TYPES' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)